images/cheat/README.md

```diff
@@ -0,0 +1,8 @@
+## General Information
+
+[Inkscape](https://inkscape.org/) should be used to edit the svg.
+Export area `Page` as a `1024x728px` png at `96`dpi to match the existing png.
```

Comment:
Can this be scripted? https://wiki.inkscape.org/wiki/Using_the_Command_Line seems like it could be.

Reply:
It certainly can be.
I changed the line to just have the command as that is certainly easier to follow.